esrc/rescueoverview.php

```diff
@@ -473,4 +483,86 @@ function displayNotes($row, $isCoord = 0, $isSARAgent = 0)
 		}
 	}
 }
+
+/**
+ * Check killboard and update rescue status if activity detected
+ * @param unknown $data
+ */
+function checkKbActivity($data) {
+
+	// create a new database connection
+	$database = new Database();
+	$kb = new KillboardActivity($database);
+	
+	$lastCheck = strtotime($kb->getLastKillboardCheck());
+	$yesterday = strtotime("-1 day");
+	
+	$isNewCheckRequired = $yesterday > $lastCheck;
+	
+	if (!$isNewCheckRequired) {		
+		return;
```

Comment:
Would a return value status to provide process information like 'checked', 'not necessary', ... be helpful to display an information to the SAR coordinator?